src/views/SystemProfile/index.tsx

```diff
@@ -614,6 +668,14 @@ const SystemProfile = () => {
       </div>
     </MainContent>
   );
+
+  return modalProps ? (
+    <Modal title={t('System Profile')} isOpen={isOpen} closeModal={closeModal}>
+      <SystemProfileView />
+    </Modal>
```

Comment:
I think we can modify the EASI `<Modal />` props to allow to configure and optional `shouldCloseOnOverlayClick` on `ReactModal` so the modal can close when clicked outside